### PR TITLE
Adding devops and acs modules to cli.

### PR DESCRIPTION
**Pull request by sanar-microsoft**
Commands added:
1. acs dcos browse: to create ssh proxy to the acs cluster.
2. devops release create: to build and deploy services to acs cluster.
3. devops release list: to list current releases in the acs cluster.
4. devops build create: to build from a repo in the acs cluster.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@sanar-microsoft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sanar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/acs_client.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        sftp = self.transport.open_sftp_client()
+        try:
+            sftp.lstat(remote_file)
+            sftp.chmod(remote_file, mode)
+        except OSError as e:
+            import errno
+            if e.errno == errno.ENOENT:
+                raise Exception('Remote file "{}" does not exist'.format(remote_file))
+            else:
+                assert False, 'Error accessing remote file {}'.format(remote_file)
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/acs_client.py]: Why would you need to assert False here

Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/acs_client.py]: Yes that sounds better than using assert.

**Comment by derekbekoe** [comment]
Merging PR.

**Pull request merged**